storage/src/main/java/org/apache/kafka/server/log/remote/storage/RemoteLogManagerConfig.java

```diff
@@ -109,6 +109,10 @@ public final class RemoteLogManagerConfig {
     public static final String REMOTE_LOG_MANAGER_TASK_RETRY_BACK_OFF_MS_DOC = "The initial amount of wait in milliseconds before the request is retried again.";
     public static final long DEFAULT_REMOTE_LOG_MANAGER_TASK_RETRY_BACK_OFF_MS = 500L;
 
+    public static final String REMOTE_LOG_MANAGER_TASK_INITIAL_DELAY_JITTER_PROP = "remote.log.manager.task.initial.delay.jitter";
+    public static final String REMOTE_LOG_MANAGER_TASK_INITIAL_DELAY_JITTER_DOC = "Initial delay when scheduling a RLMTask for the first time on leadership change";
+    public static final long DEFAULT_REMOTE_LOG_MANAGER_TASK_INITIAL_DELAY_JITTER = 0L;
```

Comment:
are we sure we want the default to be 0?

Reply:
I want the default behaviour to be the mainlline behaviour for now